db/rdbms/deckFunctions_test.go

```diff
@@ -47,6 +49,75 @@ func TestRepository_SaveDeck(t *testing.T) {
 	}
 }
 
+func TestRepository_ModifyDeck(t *testing.T) {
```

Comment:
This test is kind of redundant with the SaveDeck test, but I couldn't think of a better way to catch the bug than to first save a deck, then try to save over it (which simulates doing a `PUT` request with how the API is set up now).